beat/heart.py

```diff
@@ -2254,7 +2319,71 @@ def get_phase_arrival_time(engine, source, target, wavename=None, snap=True):
         deltat = 1. / store.config.sample_rate
         atime = trace.t2ind(atime, deltat, snap=round) * deltat
     return atime
+##Mahdi
+arrival_catalog = {'p': ('any_P', 'Z'),
+                   'sv':('any_SV', 'ZR'),
+                   'sh':('any_SH', 'T')}
+class PolarityDataset(object):
+    
+        
+    def __init__(self, store, polarityconfig, data_path):
```

Comment:
the store input here is not needed, as you hand it in when updating targets ... this is dataset! please do not mix synthetics with data

Reply:
How am I supposed to put store_id into targets?

Reply:
store_id is already an attribute of targets,
you could/ should do it as with the init_seismic_targets function ...

Reply:
I didn't use engine.get_store(target.store_id), I just put engine.get_store().